stable/jenkins/templates/config.yaml

```diff
@@ -94,7 +94,14 @@ data:
                   </envVars>
                 </org.csanchez.jenkins.plugins.kubernetes.ContainerTemplate>
               </containers>
-              <envVars/>
+              <envVars>
+{{- range $index, $var := .Values.Agent.envVars }}
```

Comment:
it is missing the envVars in the values.yaml

Reply:
True, I'm adding it